doc/schemas/app_openapi.json

```diff
@@ -238,12 +238,26 @@
         "scheme": "signature",
         "type": "http"
       }
+    },
+    "x-ccf-forwarding": {
+      "always": {
+        "description": "If this request is made to a backup, it will be forwarded to a primary node for execution. Should be used for operations which may produce writes.",
```

Comment:
```suggestion
        "description": "If this request is made to a backup node, it will be forwarded to the primary node for execution. Should be used for operations which may produce writes.",
```

Reply:
"Should be used for operations which may produce writes." reads like something I'd expect to read in the documentation for building apps, but not in the OpenAPI spec for a given endpoint, once the app has been built?

Reply:
Good point. I've moved these sections to a comment on the enum (which should be in the docs), and removed it from the OpenAPI.